app/src/app.config.json.tpl

```diff
@@ -6,6 +6,13 @@
   "providers": "${APP_CONFIG_PROVIDER}",
   "authType": "${APP_CONFIG_AUTH_TYPE}",
   "loginRoute": "login",
+  "mobileAppSwitch": {
+    "enabled" : "${APP_CONFIG_ENABLE_MOBILE_APP_SWITCH}",
+    "isIphone": "iosamw://",
+    "isAndroidPart1": "intent:///",
+    "isAndroidPart2": "#Intent;scheme=androidamw;package=com.alfresco.content.app.debug;end",
```

Comment:
how do customers set their own apps here? and why it has app.debug?

Reply:
We are enforcing the user to open amw mobile app so we sets it's package and I have removed .debug from here.